src/components/OutcomeCategorical/index.js

```diff
@@ -50,11 +61,16 @@ const OutcomeCategorical = ({ market, opts = {} }) => {
             <div className="outcome__bar">
               <div
                 className="outcome__bar--inner"
-                style={{ width: `${tokenDistribution[outcomeIndex] * 100}%`, backgroundColor: COLOR_SCHEME_DEFAULT[outcomeIndex] }}
+                style={{
```

Comment:
Objects should be constants outside of the render function, otherwise, this component always will be re-rendered.